### PR TITLE
Rac help

### PR DESCRIPTION
**Pull request by LornaSIL**
* fixed help file for Rachitrali-Brahui (fixes #369)
* fixed help file for Rachitrali-Dameli
* added help file for Rachitrali-Balti

### CODE REVIEW

**Comment by rachitralli** [comment]
@LornaSIL thanks

File: release/rac/rac_balti/source/help/rac_balti.php [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+SOFTWARE.</p></DIV>
+
+
+
+<h2>Release History</h2>
+<p>Version 1.0, released 04 June 2018</p>
+
+<h2>Links</h2>
+<ul>
+<li><a href='http://keymankeyboards.com/iso:bft'>Other keyboards for these languages</a></li>
```
Comment (darcywong00) [inline release/rac/rac_balti/source/help/rac_balti.php]: Maybe we can use this link for other Balti keyboards:
https://keyman.com/keyboards/languages/bft

File: release/rac/rac_balti/source/help/rac_balti.php [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+      <P>The keyboard layout is displayed below in Keyboard graphics.</P>
+      	
+      	<h3>Base Version</h3>
+      	<img src="Base.png" alt="Base Version" style="width:600px;">
+
+      	<h3>Shift Version</h3>
+      	<img src="Shift.png" alt="Shift Version" style="width:600px;">
+      	
+      	<h3>CTRL ALT Version</h3>
+      	<img src="CtrlAlt.png" alt="Shift Version" style="width:600px;">
```
Comment (darcywong00) [inline release/rac/rac_balti/source/help/rac_balti.php]: This is `alt="Ctrl-Alt Version"`

File: release/rac/rac_balti/source/help/rac_balti.php [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+</p>
+<p style="text-align: right; dir=rtl; font-size:150%">
+بلتی سافٹ ویر اور بلتی ٹیکسٹ ایڈیٹر ایجاد کرنے کا تاریخی کام پاکستان کے ضلع چترال سے تعلق رکھنے والے لسانی ماہر جناب رحمت عزیز چترانی نے شروع کیا ہے۔ اور آج اس سافٹ ویر کے زریعے رحمت عزیز چترالی نے بلتی جیسی غیر تحریری زبان کو تحریری زبان میں تبدیل کرکے کمپیوٹر اور موبائل کی زبان بنا دیا ہے اور بلتی بولنے والے لوگ اب فیس بک اور ٹویٹر میں بھی اس سافٹ ویر کے زریعے بلتی لکھ رہے ہیں جو کہ بلتی دان طبقے کے لیے یقینا بہت بڑی خوشی کی بات ہے
+</p>
+
+      <P class=Caption></P><A name=license></A>
+      <H2>License</H2>
+      <P>Rachitrali-Balti Keyman Keyboard is copyright © 2018 Rehmat Aziz Chitrali. 
+      	It is distributed under The MIT License (MIT)</P>
+      <DIV class=Note>
```
Comment (darcywong00) [inline release/rac/rac_balti/source/help/rac_balti.php]: to apply the "note" styling
```
<div class="note">
```

File: release/rac/rac_brahui/source/help/rac_brahui.php [hunk 4]
```diff
@@ -1,55 +1,111 @@
...
+
+
+
+<h2>Release History</h2>
+<p>Version 1.0, released 03 June 2018</p>
+
+<h2>Links</h2>
+<ul>
+<li>Home Page: <a href='http://keymankeyboards.com'>http://keymankeyboards.com/</a></li>
+<li><a href='http://keymankeyboards.com/iso:brh'>Other keyboards for these languages</a></li>
```
Comment (darcywong00) [inline release/rac/rac_brahui/source/help/rac_brahui.php]: this link can be https://keyman.com/keyboards/languages/brh

File: release/rac/rac_dameli/source/help/rac_dameli.php [hunk 5]
```diff
@@ -114,5 +112,4 @@
 </ul>
```
Comment (darcywong00) [inline release/rac/rac_dameli/source/help/rac_dameli.php]: link on line 111 can be https://keyman.com/keyboards/languages/dml

**Comment by darcywong00** [comment]
It looks like the CI server build fails because the .kps files reference the RAC.jpg files

File: release/rac/rac_dameli/source/help/rac_dameli.php (hunk 5 above)
Comment (rachitralli) [reply release/rac/rac_dameli/source/help/rac_dameli.php]: @Darcywong00 the mobile image is missing in the link. Pls add

**Comment by rachitralli** [comment]
@LornaSIL thanks

**Pull request merged**